File: app/src/main/java/com/example/linqing/zhihumvp/detail/DetailContract.java
package com.example.linqing.zhihumvp.detail;

import com.example.linqing.zhihumvp.entitybean.ArticleDetailBean;

public interface DetailContract {
    interface DetailView{
        void loadWebDetail(ArticleDetailBean data);


    }

    interface DetailPresenterInter{

        void getWebviewData();
        void getIdP(String id);
    }
}

File: app/src/main/java/com/example/linqing/zhihumvp/detail/DetailContract.kt
package com.example.linqing.zhihumvp.detail

import com.example.linqing.zhihumvp.entitybean.ArticleDetailBean

interface DetailContract {
    interface DetailView {
        fun loadWebDetail(data: ArticleDetailBean)


    }

    interface DetailPresenterInter {

        fun getWebviewData()
        fun getIdP(id: String)
    }
}
